features/support/recommended_links.rb

```diff
@@ -118,3 +118,13 @@ def build_doc_from_recommended_link(recommended_link, include_es_data: false)
     details
   end
 end
+
+private
+
+def type(recommended_link)
```

Comment:
I think it's necessary at the moment because we have cucumber steps like this:

```
Then(/^all recommended links should have been exported in the correct ElasticSearch format$/) do
  confirm_recommended_links_elasticsearch_format(@exporter_output, RecommendedLink.all)
end
```

Which is part of this scenario:

```
  Scenario: Exporting a variety of recommended links
    Given a variety of recommended links exist
    When I run recommended links ElasticSearch exporter
    Then all recommended links should have been exported in the correct ElasticSearch format
```

Do you think it's worth changing this to be more explicit?